docs/reference/snapshot-restore/apis/get-snapshot-status-api.asciidoc

```diff
@@ -0,0 +1,228 @@
+(boolean)
+Indicates whether the current cluster state is included in the snapshot.
+
+`shards_stats`::
+(object)
+Contains a count of shards in the snapshot.
++
+.Properties of `shards`
+[%collapsible%open]
+====
+`initializing`::
+(integer)
+Number of shards that are still initializing.
+
+`started`::
+(integer)
+Number of shards that have started but not are not finalized.
+
+`finalizing`::
+(integer)
+Number of shards that are finalizing but are not done.
+
+`done`::
+(integer)
+Number of shards that initialized, started, and finalized successfully.
+
+`failed`::
+(integer)
+Number of shards that failed to be included in the snapshot.
+
+`total`::
+(integer)
+Total number of shards included in the snapshot.
+====
+
+`stats`::
+(object)
+Provides details on the number (`file_count`) and size (`size_in_bytes`) of files included in the snapshot.
++
+.Properties of `stats`
+[%collapsible%open]
+====
+`incremental`::
+(object)
+Number and size of files that are not already in the repository and need to be copied as part of the incremental snapshot.
+
+`processed`::
+(object)
+Number and size of files that are still being copied.
+
+`total`::
+(object)
+Total number and size of files that are referenced by the snapshot.
+
+`start_time_in_millis`::
+(long)
+The time, in milliseconds, when the snapshot creation process started.
+
+`time_in_millis`::
+(long)
+The total time, in milliseconds, that
+====
+
+`indices`::
+(list of objects)
+List of objects containing information about the
+indices included in the snapshot.
++
+Detailed information for each index includes:
+
+* `shards_stats`
+* `stats`
+* `shards`
+
+[[get-snapshot-status-api-example]]
+==== {api-example-title}
+
+The following request returns information for `snapshot_2` in the `my_repository` repository.
+
+[source,console]
+----
+GET /_snapshot/my_repository/snapshot_2/_status
+----
+
+The API returns the following response:
+
+[source,console-result]
+----
+{
+  "snapshots" : [
+    {
+      "snapshot" : "snapshot_2",
+      "repository" : "my_repository",
+      "uuid" : "ITWHQY2fR2OqX-TYt6mUNA",
+      "state" : "SUCCESS",
+      "include_global_state" : true,
+      "shards_stats" : {
+        "initializing" : 0,
+        "started" : 0,
+        "finalizing" : 0,
+        "done" : 0,
+        "failed" : 0,
+        "total" : 0
+      },
+      "stats" : {
+        "incremental" : {
+          "file_count" : 0,
+          "size_in_bytes" : 0
+        },
+        "total" : {
+          "file_count" : 0,
+          "size_in_bytes" : 0
+        },
+        "start_time_in_millis" : 1594158151205,
+        "time_in_millis" : 0
+      },
+      "indices" : { }
+    }
+  ]
+}
+----
+// TESTRESPONSE[s/"uuid": "ITWHQY2fR2OqX-TYt6mUNA"/"uuid": $body.snapshots.0.uuid/]
+// TESTRESPONSE[s/"start_time_in_millis": 1594158151205/"start_time_in_millis": $body.snapshots.0.start_time_in_millis/]
+// TESTRESPONSE[s/"time_in_millis": 0/"time_in_millis": $body.snapshots.0.time_in_millis/]
```

Comment:
Couple of errors here:
*  The response includes a space between the property name and the `:`. However your comments don't include this space. To fix, change either so they're consistent.
* The `start_time_in_millis` and `time_in_millis` properties are nested in the `stats` property. You'll need to add this property to your respective comments.

```suggestion
// TESTRESPONSE[s/"uuid" : "ITWHQY2fR2OqX-TYt6mUNA"/"uuid" : $body.snapshots.0.uuid/]
// TESTRESPONSE[s/"start_time_in_millis" : 1594158151205/"start_time_in_millis" : $body.snapshots.0.stats.start_time_in_millis/]
// TESTRESPONSE[s/"time_in_millis" : 0/"time_in_millis" : $body.snapshots.0.stats.time_in_millis/]
```

Reply:
Thank you @jrodewig -- I originally had `.stats` in the test but removed it because I thought that it could be breaking the CI tests. However, just missing spaces 😆